glide.lock

```diff
@@ -173,8 +173,12 @@ imports:
   version: a0d98a5f288019575c6d1f4bb1573fef2d1fcdc4
   subpackages:
   - simplelru
+- name: github.com/howeyc/gopass
```

Comment:
It is being pulled from the k8s apiextension project. https://github.com/kubernetes/apiextensions-apiserver/blob/master/Godeps/Godeps.json#L232